bin/cli/sqs.rb

```diff
@@ -158,11 +158,11 @@ def requeue(queue_name, path)
       desc 'mv QUEUE-NAME-SOURCE QUEUE-NAME-TARGET', 'Move messages from one queue (source) to another (target)'
       method_option :number, aliases: '-n', type: :numeric, default: Float::INFINITY, desc: 'number of messages to move'
       method_option :delete, aliases: '-d', type: :boolean, default: true,            desc: 'delete from the queue'
-      def mv(queue_name_source, queue_name_target)
+      def mv(queue_name_source, queue_name_target, limit: nil)
```

Comment:
@parov if feels like hacky doing that, it's a thor task, it's supposed to be called as a thor task. I wouldn't add an extra param, just for making it available for outside, the possible options:

1. Invoke is a thor task. See https://github.com/erikhuda/thor/wiki
2. subprocess `system(bundle exec shoryuken sqs mv ...)`
3. Extract this MV to a class (not a thor one) and use this class here and in your controller.